akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/internal/ClusterShardingImpl.scala

```diff
@@ -305,8 +302,7 @@ import akka.util.Timeout
 @InternalApi private[akka] final class EntityRefImpl[M](
     shardRegion: akka.actor.ActorRef,
     entityId: String,
-    typeKey: EntityTypeKeyImpl[M],
-    scheduler: Scheduler)
+    typeKey: EntityTypeKeyImpl[M])
```

Comment:
I removed scheduler here as it wasn't used, we can add it back later if we need it